### DISCUSSION TITLE
torch_geometric.explain.Explainer giving ValueError: 'multiclass_classification' is not a valid ModelMode

### DISCUSSION THREAD

**Discussion by yangyangdotcom** [pyg-team/pytorch_geometric#6783] [Q&A] [answered]
I was trying to replica the result of [this](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gnn_explainer.py). However, I encounter a ValueError message stating that the ModelMode I choose is invalid. However, according to [this](https://pytorch-geometric.readthedocs.io/en/latest/modules/explain.html#torch_geometric.explain.config.ModelConfig), multiclass_classification indeed is a valid ModelMode. Any clue why it does not work on 'multiclass_classification'? Thanks in advance!

**Comment by wsad1** [comment, marked as answer]
Apologies for this. Which version of pyg are you an, can you try installing from master pip install git+https://github.com/pyg-team/pytorch_geometric.git. The example is written assuming master is used.

**Reply by yangyangdotcom** [reply]
Thank you! It worked!
